ButtplugTest/Core/ButtplugJsonMessageParserTests.cs

```diff
@@ -10,9 +10,10 @@ public class ButtplugJsonMessageParserTests
         public void JsonConversionTest()
         {
             var m = new Buttplug.Messages.Test("ThisIsATest", ButtplugConsts.SYSTEM_MSG_ID);
-            var msg = ButtplugJsonMessageParser.Serialize(m);

```

Comment:
Here and below, we can just bring up a ButtplugServer here and use serialize through that, versus just bringing up a logger.